paths/electron/starting-with-electron/08-add-to-js.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Add JavaScript
+permalink: /electron/starting-with-electron/add-to-javascript
+next-page: /electron/starting-with-electron/celebrate
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  Finally, let's add the fun part: the Javascript!
+
+  1. Copy the Javascript from Codepen into the `src/script.js` file.
+  1. Spin up your app, and see how it works.
+  1. Save and commit your changes.
```

Comment:
@hollenberry Thank you for taking the time to define the bug and find a fix. :heart: I think we should [discuss in our standup today](https://github.com/github/services-training/issues/539#issuecomment-321253037) if we want to choose to A) use a very simple project that won't require jQuery OR B) just include these instructions.